pages/conduct/reporting.md

```diff
@@ -0,0 +1,50 @@
+<script>{
+	"title": "jQuery Foundation Code of Conduct - Reporting Guide",
+	"pageTemplate": "page-conduct.php"
+}</script>
+
+If you believe someone is violating the code of conduct we ask that you report it to the jQuery Foundation by emailing [conduct@jquery.com](mailto:conduct@jquery.com). **All reports will be kept confidential.** In some cases we may determine that a public statement will need to be made. If that's the case, the identities of all victims and reporters will remain confidential unless those individuals instruct us otherwise.
+
+**If you believe anyone is in physical danger, please notify appropriate law enforcement first.** If you are unsure what law enforcement agency is appropriate, please include this in your report and we will attempt to notify them.
+
+In your report please include:
+
+*   Your contact info (so we can get in touch with you if we need to follow up)
+*   Names (real, nicknames, or pseudonyms) of any individuals involved. If there were other witnesses besides you, please try to include them as well.
+*   When and where the incident occurred. Please be as specific as possible.
+*   Your account of what occurred. If there is a publicly available record (e.g. a mailing list archive or a public IRC logger) please include a link.
```

Comment:
It would be a good idea to have them preserve a copy/screenshot of the evidence in case it is deleted.